FluentTerminal.SystemTray/Utilities.cs

```diff
@@ -569,44 +569,58 @@ internal static void SaveFile(string path, string content)
             }
         }
 
-        internal static void MuteTerminal(bool mute)
+        private static bool MuteConhost(bool mute, int attemptCount)

```

Comment:
Why are there different `attemptCount` values for different call sites?

Reply:
have removed looping completely